src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1099,13 +1099,19 @@ private bool ShouldSkipCommandResolutionForConstrainedLanguage(CommandInfo? resu
                 // Relative Path:       ".\command.exe"
                 // Home Path:           "~\command.exe"
                 // Drive Relative Path: "\Users\User\AppData\Local\Temp\command.exe"
-                if (!string.IsNullOrEmpty(_commandName) && (_commandName[0] == '.' || _commandName[0] == '~' || _commandName[0] == '\\'))
+
+                //char firstChar = _commandName[0];
+                if (_commandName.Length != 0) 
```

Comment:
Please remove commented code and move the condition in line 1104 to line 1090.

Reply:
Oh yeah thanks. Can't believe I missed that.